2. Services/2.0 Foundations/2.0 Foundations.md

```diff
@@ -41,7 +41,7 @@ Here's the charactristics and rules that govern broker-neighboring services:
 ### 2.1.2.0 Pure-Primitive
 Broker-neighboring services are not allowed to combine multiple primitive operations to achieve a higher-order business logic operation.
 
-For instance, broker-neighboring services cannot offer an *upsert* function, to combine a `Select` operations with an `Update` or `Insert` operations based on the outcome to ensure an entity exists and is update to date in any storage.
+For instance, broker-neighboring services cannot offer an *upsert* function, to combine a `Select` operations with an `Update` or `Insert` operations based on the outcome to ensure an entity exists and is up to date to date in any storage.
```

Comment:
We could just say `and is up to date in any storage` 
What do you think about that?

Reply:
yup.. that's better

Reply:
I updated the commit.